### PR TITLE
mount: umount on SIGINT/^C when in foreground

### LINKED ISSUE #2082: borg mount -f should umount when SIGINT (ctrl-c)
Currently, the mount remains beyond the command execution.
This behaviour is inconvenient for those who use it as a means of assuring you don't forget to umount it (and thus remove the lock).

Thanks!

### PR DESCRIPTION
**Pull request by enkore**
+document it

Fixes #2082

### CODE REVIEW

**Comment by enkore** [comment]
While #2094.

File: borg/archiver.py [hunk 1]
```diff
@@ -1546,6 +1546,13 @@ def build_parser(self, args=None, prog=None):
         - allow_damaged_files: by default damaged files (where missing chunks were
           replaced with runs of zeros by borg check --repair) are not readable and
           return EIO (I/O error). Set this option to read such files.
+
+        When the daemonized process receives a signal or crashes it does not unmount
```
Comment (ThomasWaldmann) [inline borg/archiver.py]: add a dot at the end of the sentence.
should there be a comma after "crashes"?

**Comment by ThomasWaldmann** [comment]
this could be finished and merged rather quickly, there is only a minor todo left.

**Comment by enkore** [comment]
Oh, I thought I already merged it. Done.

**Pull request merged**